noaa-cpfp-point/src/notice/index.js

```diff
@@ -0,0 +1,30 @@
+const chartContainer = document.getElementById("chart-container");
+const noticeContainer = document.getElementById("notice-container");
+const noticeContent = document.getElementById("notice-content");
+const mapContainer = document.getElementById("map-container");
+
+export function openNotice(text) {
```

Comment:
do we need a bigger container or more padding at the bottom?
![image](https://github.com/US-GHG-Center/custom-interfaces/assets/7830949/9df661b7-1697-4b9f-a719-6135f63f6239)

Reply:
made the container bigger by 1%. space at foot lgtm!

![image](https://github.com/US-GHG-Center/custom-interfaces/assets/38744044/528ab5e2-b591-485f-910b-af6cd9142e70)